### ISSUE TITLE
input and output information

### ISSUE THREAD

**Issue by coraglio** [Anuken/Mindustry-Suggestions#4533]
### Describe what you would like changed, and why.

It takes a lot of time to look at the description of each building and know what amounts of input and output they have of each item. 

### Describe the changes you want to propose. Include possible alternatives.

They must add that information in the box that appears when the building is selected.

**Comment by Lime712** [comment]
💀☠️🏴‍☠️🪦⚰️🗿🕱

**Comment by itcannotbe** [comment]
Learning curve is a thing. You will find those things not of help after a few hours of actual gameplay.

**Comment by TheEt1234** [comment]
press the [?] button in the block menu

**Comment by coraglio** [comment]
Exactly, you have to search and touch the [?] button, that's what I want to avoid, because to create complex structures I'm going to write down in a notebook what the inputs and outputs of each component are. And I think the learning curve is interesting, but from the place where you have to think about the most efficient way to do things, but it seems unnecessary to me to have to memorize each value.

**Comment by buthed010203** [comment]
> Exactly, you have to search and touch the [?] button, that's what I want to avoid, because to create complex structures I'm going to write down in a notebook what the inputs and outputs of each component are. And I think the learning curve is interesting, but from the place where you have to think about the most efficient way to do things, but it seems unnecessary to me to have to memorize each value.

Alternatively, just press f1 while hovering a block
